src/applications/personalization/profile-2/components/direct-deposit/DirectDepositContent.jsx

```diff
@@ -42,44 +42,38 @@ export const DirectDepositContent = ({
   const editBankInfoButton = useRef();
   const [formData, setFormData] = useState({});
   const [showSaveSucceededAlert, setShowSaveSucceededAlert] = useState(false);
-  const previousUiState = usePrevious(directDepositUiState);
+  const wasEditingBankInfo = usePrevious(directDepositUiState.isEditing);
```

Comment:
Rather than saving the entire previous UI state I switched to more granular vars so that the useEffect hooks that use them are a little bit easier to understand. I think it's easier to understand `wasSavingBankInfo` vs. `previousUiState?.isSaving`, for example.